WordPress/Classes/Services/CommentService.m

```diff
@@ -1136,46 +1136,15 @@ - (Comment *)findCommentWithID:(NSNumber *)commentID fromPost:(ReaderPost *)post
 #pragma mark - Transformations
 
 - (void)updateComment:(Comment *)comment withRemoteComment:(RemoteComment *)remoteComment
-{
-    comment.commentID = remoteComment.commentID;
-    comment.author = remoteComment.author;
-    comment.author_email = remoteComment.authorEmail;
-    comment.author_url = remoteComment.authorUrl;
-    comment.authorAvatarURL = remoteComment.authorAvatarURL;
-    // Ref: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/413
-    // Use raw_content for blog comments so
-    // This is a temporary mixmatch of properties for a frozen branch bug fix that we'll sort out properly in develop.
-    // - @aerych
-    comment.content = remoteComment.rawContent;
-    comment.dateCreated = remoteComment.date;
-    comment.link = remoteComment.link;
-    comment.parentID = remoteComment.parentID;
-    comment.postID = remoteComment.postID;
-    comment.postTitle = remoteComment.postTitle;
-    comment.status = remoteComment.status;
-    comment.type = remoteComment.type;
-    comment.isLiked = remoteComment.isLiked;
-    comment.likeCount = remoteComment.likeCount;
-    comment.canModerate = remoteComment.canModerate;
-
-    // if the post for the comment is not set, check if that post is already stored and associate them
-    if (!comment.post) {
-        PostService *postService = [[PostService alloc] initWithManagedObjectContext:self.managedObjectContext];
-        comment.post = [postService findPostWithID:comment.postID inBlog:comment.blog];
-    }
-}
-
-// Temporary duplication of [updateComment:withRemoteComment:] to address an issue in a frozen branch
-// We'll sort it out properly in develop.
-// - @aerych
-- (void)updateReaderComment:(Comment *)comment withRemoteComment:(RemoteComment *)remoteComment
 {
     comment.commentID = remoteComment.commentID;
     comment.author = remoteComment.author;
     comment.author_email = remoteComment.authorEmail;
     comment.author_url = remoteComment.authorUrl;
     comment.authorAvatarURL = remoteComment.authorAvatarURL;
     comment.content = remoteComment.content;
+    // rawContent contains HTML for P2 comments. Remove it to get true raw content.
+    comment.rawContent = [[[[remoteComment.rawContent stringByDecodingXMLCharacters] trim] stringByStrippingHTML] stringByNormalizingWhitespace];
```

Comment:
> It might not matter until we get to the block editor, just thinking ahead.

Ah, I didn't think about that. I'll move the filtering back to `Comment`.

> I notice that with this patch I'm seeing html when editing a comment

Ugh. I thought I checked that. Thank you!